### ISSUE TITLE
Add kinship data

### ISSUE THREAD

**Issue by SimonGreenhill** [concepticon/concepticon-data#844]
In order to link kinbank to the concepticon, it would be nice to have the extended kinship list added: 

https://github.com/SimonGreenhill/kinbank/blob/master/kinbank/cldf/parameters.csv

Note that there's no publication on this, but there is one on the way (I'm not sure on the timeframe of this -- @sampassmore ?). 

The main issue is what to do with EGO (i.e.  “the person speaking”). We currently have this mapping e.g.

```
fDH	female ego's daughter's husband	2266	SON-IN-LAW (OF WOMAN)
```

… so we could use the “(OF X)” where X = {WOMAN, MAN, FATHER, …} , and generate all the others? 

Although I wonder if “OF WOMAN” is confusing. I’m taking it to mean how a woman refers to her SON-IN-LAW, but I can see it getting confused with the SON-IN-LAW you have from a woman (e.g. your wife’s son from a previous marriage?)

We also need to add the conceptrelations too (e.g. `female ego's daughter's husband` is a narrower instanceof `1056 SON-IN-LAW`).

**Comment by LinguList** [comment]
We can also adjust our terms in concepticon, it would just mean we have to make sure to retire them properly, and it will mean we have to change many glosses, and it contradicts the practice we find in most lists (even if that practice is not okay).

**Comment by LinguList** [comment]
These kinshipt terms are hard for me, so I would suggest that somebody more versed makes a good proposal for naming in concepticon, so that we have this covered. Changing glosses in concepticon is also not problematic, I am just afraid that this "ego" will confuse too much (so having the ego-part in the explanation may also generally be easier).

**Comment by LinguList** [comment]
> Although I wonder if “OF WOMAN” is confusing. I’m taking it to mean how a woman refers to her SON-IN-LAW, but I can see it getting confused with the SON-IN-LAW you have from a woman (e.g. your wife’s son from a previous marriage?)

There are problems already, we discussed this in the past, @Kristina-Pianykh, I think noticed inconsisttencies, and I told in that thread that we may wait for kinbank to solve things...

**Comment by SimonGreenhill** [comment]
yeah, I don't want to make a major set of changes here, so keeping it minimal is the best thing. This might be a case where we just leave 95% of the concepts unlinked (i.e. only add the ones that are in concepticon already?)

**Comment by LinguList** [comment]
Also possible. You could also say: kinbank is THE reference catalog for kinshipt terms, and whenever concepticon finds one, we link to kinbank in our metadata (!). So we add a mapping from concepticon concept sets to a reference in kinbank (maybe the letter code)? And we steal the definitions, this would help us to get rid of inconsistencies there.

**Comment by chrzyki** [comment]
> Also possible. You could also say: kinbank is THE reference catalog for kinshipt terms, and whenever concepticon finds one, we link to kinbank in our metadata (!). So we add a mapping from concepticon concept sets to a reference in kinbank (maybe the letter code)? And we steal the definitions, this would help us to get rid of inconsistencies there.

A good plan I think. I also agree with moving the EGO part to the definition. I'll have a chat with @Kristina-Pianykh to plan this. Should probably involve multiple step-by-step PRs to get this done consistently and in a reproducible fashion (to avoid any mishaps in Concepticon and the datasets relying on it).

**Comment by SimonGreenhill** [comment]
@supertyp

**Comment by SamPassmore** [comment]
RE: Kinbank publication 
I wouldn't anticipate it before the end of summer (because I need to write my thesis), but if you need a place holder publication there should be something coming out in Biological Theory soon. Can keep you posted. 

Let me know if there is anything I can help with!
